chaining/challenges-10.test.js

```diff
@@ -0,0 +1,190 @@
+
```

Comment:
Make sure to check out `master` before you check out the new branch, so you don't get old files showing up in here!